### PR TITLE
css changes

### PR DESCRIPTION
**Pull request by iatkins**

### CODE REVIEW

File: src/components/PlayerInfo/PlayWordAndCard/PlayWordAndCard.module.css [hunk 1]
```diff
@@ -22,30 +19,41 @@ h1 {
...
 .playerInteractions > h3 {
     color: var(--lighter-black);
     margin-right: 1em;
     text-align: center;
     width: 100%;
-    font-size: 5vmin;
+    font-size: 3.5vmin;
+    margin-bottom: 0.5em;
+    
 }
```
Comment (tillywoodfield) [inline src/components/PlayerInfo/PlayWordAndCard/PlayWordAndCard.module.css]: New line

File: src/components/PlayerInfo/VoteCard/VoteCard.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import styles from '../PlayWordAndCard/PlayWordAndCard.module.css';
 
 export const VoteCard = () => {
     return (
-        <div className={styles.playerInteractions}>
+        <div className={styles.playerInteractionsting}>
```
Comment (tillywoodfield) [inline src/components/PlayerInfo/VoteCard/VoteCard.jsx]: Is this a typo?

File: src/components/Dashboard/Dashboard.module.css [hunk 3]
```diff
@@ -36,4 +36,222 @@ h3 {
...
+    to { transform: translateY(4%); } 
+  }
+  
+  @keyframes bouncebody { 
+    to { transform: scaleX(1.03) scaleY(0.97); } 
+  }
+  
+  @keyframes antena_d { 50% {transform: rotate(30deg);} }
+  @keyframes antena_e { 50% {transform: rotate(-40deg);} }
+
```
Comment (tillywoodfield) [inline src/components/Dashboard/Dashboard.module.css]: New line

**Review by Ro4052 — COMMENTED** [review]
Looks fine but you might want to move this stuff into its own component.

File: src/components/Dashboard/Dashboard.module.css [hunk 4]
```diff
@@ -36,4 +36,222 @@ h3 {
...
+.ufo {
+    font-size: 25vmin;
+    position:absolute;
+    width: 10%;
+    height:10%;
+    bottom: 0;
+    right: 0;
+  }
+  
+  .monster {
```
Comment (tillywoodfield) [inline src/components/Dashboard/Dashboard.module.css]: Randomly indented

File: src/components/PlayerInfo/VoteCard/VoteCard.jsx (hunk 2 above)
Comment (iatkins) [reply src/components/PlayerInfo/VoteCard/VoteCard.jsx]: no i needed to detach it from player interactions so i added ting on the end but can change it to something more descriptive

File: src/components/Dashboard/Dashboard.module.css [hunk 5]
```diff
@@ -36,4 +36,222 @@ h3 {
...
+    box-sizing: content-box;
+    border: 0.07em solid transparent;
+    position: absolute;
+    bottom:0em;
+    user-select: none;
+    cursor: grab;
+    right: 0;
+  }
+  
+  .body {
```
Comment (tillywoodfield) [inline src/components/Dashboard/Dashboard.module.css]: Same with all of this

File: src/components/Dashboard/Dashboard.jsx [hunk 6]
```diff
@@ -34,10 +34,26 @@ export class Dashboard extends React.Component {
                 <Players />
                 <PlayedCards />
                 <PlayerCards />
-                {this.props.playWordAndCard && <PlayWordAndCard />}
+                {this.props.playWordAndCard && !this.props.finishedRound && <PlayWordAndCard />}
                 {this.props.playCard && <PlayCard />}
                 {this.props.voteCard && this.props.votedCard === 0 && <VoteCard />}
                 <LogoutButton />
+                <div className={style.ufo}>
+                        <div className={style.monster}>
```
Comment (tillywoodfield) [inline src/components/Dashboard/Dashboard.jsx]: Extra indent here

File: src/components/Dashboard/Dashboard.module.css [hunk 7]
```diff
@@ -36,4 +36,222 @@ h3 {
...
+    width: 100%;
+    height: 100%;
+    background-color: currentColor;
+    border-radius: inherit;
+    position: relative;
+    transform-origin: bottom center;
+    animation: bouncebody alternate infinite 400ms 40ms ease-in-out;
+  }
+  
+  .eyes, .eyelid {
```
Comment (tillywoodfield) [inline src/components/Dashboard/Dashboard.module.css]: More random indenting here

File: src/components/Login/Login.module.css [hunk 8]
```diff
@@ -173,6 +175,7 @@ div.foo {
 }
 
 .monster {
```
Comment (tillywoodfield) [inline src/components/Login/Login.module.css:177]: Can you change these to be 4 spaces indent

File: src/components/PlayerInfo/VoteCard/VoteCard.jsx (hunk 2 above)
Comment (tillywoodfield) [reply src/components/PlayerInfo/VoteCard/VoteCard.jsx]: The `styles.playerInteractions` was probably adding other styles so by changing the name you might break things that Sisi has done with the css, if you want to add a new class for your own styling then you should make another class with a descriptive name and add that as well, can show you how if you struggle

**Review by tillywoodfield — APPROVED** [review]

**Pull request merged**